errbot/config-template.py

```diff
@@ -93,6 +93,12 @@
 # For example CORE_PLUGINS = ('ACLs', 'Backup', 'Help') you get those names from the .plug files Name entry.
 # For absolutely no plug: CORE_PLUGINS = ()
 
+# Defines an order in which the plugins are getting their callbacks. Useful if you want to have plugins do
+# pre- or post-processing on messages.
+# The 'None' tuple entry represents all the plugins that aren't to be expicitely ordered. For example, if
```

Comment:
The None parameter is so you can have plugins that are to be executed before, and plugins that are to be executed after. If I remove it, you can only have plugins that are executed before everything else, and no longer have plugins that are executed after everything else.